modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -12,51 +12,32 @@ If you plan to install Mule and run it on premises, review these minimum hardwar
 
-If you use an application server or need to access a data store, verify that you use a supported application server or database.
+Mule Runtime runs as a standalone server; therefore an application server is not required. However, if you use an application server or need to access a data store, verify that you use a supported application server or database.
 
-=== Supported Java Versions
+[%header,cols="15a,85a"]
+|===
+|Software |Version
+| JDK | JDK 1.8.0 (JDK 1.8.0_161 recommended), OpenJDK 8 (JDK 1.8.0_162 recommended), IBM JDK 1.8
+| OS | MacOS 10.11.x, HP-UX 11i V3, AIX 7.2, Windows 2016 Server (recommended), Windows 2012 R2 Server, Windows 10 (recommended), Windows 8.1, Solaris 11.3, RHEL 7, Ubuntu Server 16.04, or any other Linux distribution that runs a supported JDK and Linux kernel 3.13
+| Application Servers | Tomcat 7, Tomcat 8, Weblogic 12c, Wildfly 8, Wildfly 9, Websphere 8, Jetty 8, Jetty 9
+| Databases | Oracle 11g, Oracle 12c, MySQL 5.5+, IBM DB2 10, PostgreSQL 9, Derby 10, Microsoft SQL Server 2014
+| Browsers for working on Anypoint Platform | The latest versions of Firefox, Chrome, and Edge. Note that IE is being deprecated; minimum screen resolution of 1024x768
+|===
 
-* Oracle JDK 1.8.0 (JDK 1.8.0_161 recommended)
-* IBM JDK 1.8
-* OpenJDK 8 (JDK 1.8.0_162 recommended)
+If you use the Microsoft Dynamics GP connector, install Java Cryptography Extension (JCE) 8 after installing JRE or JDK.
 
-=== Supported Operating Systems
+The unified Mule Runtime 4.1.5 and API Gateway is compatible with APIkit 1.1.8.
 
-* MacOS 10.11.4
-* HP-UX 11i V3
-* AIX 7.2
-* Windows Server 2016 (recommended), Windows 2012 R2 Server, Windows 10 (recommended), or Windows 8.1
-* Solaris 11.3
-* RHEL 7
-* Ubuntu Server 15.04 or 16.04, or any other Linux distribution that runs a supported JDK and Linux kernel 3.13
+This version of Mule Runtime is bundled with the Runtime Manager Agent plugin version 2.1.9.
 
-=== Application Servers
-
-An application server is not required because Mule runs as a standalone server. However, Mule can run under one of these supported application servers:
-
-* Tomcat 7 and 8
-* WebLogic 12c
-* WebSphere 8
-* WildFly 8 or 9
-* Jetty 8 or 9
-
-=== Additional Requirements
-
-* If you use the Microsoft Dynamics GP connector, install Java Cryptography Extension (JCE) 8 after installing JRE or JDK.
-* If you need to access a data store, a database is required. Mule can access any of the following supported databases:
-+
-** Mule Oracle 11g
-** Oracle 12c
-** MySQL 5.5+
-** IBM DB2 10
-** PostgreSQL 9
-** Derby 10
-** Microsoft SQL Server 2014
+Anypoint Connector DevKit 3.9.5 is compatible with Mule Runtime versions 4.1 and later. DevKit version numbering does not correspond to Mule versions. For example, DevKit 3.9 is compatible with Mule versions 3.5-4.1. For the latest fixes and improvements, use the current version of Mule Runtime with its corresponding DevKit as listed in release notes.
```

Comment:
This is wrong, DevKit has nothing to do with Mule 4, I see you refer to release notes, is this coming from another docs page? We should remove this completely.

Reply:
Removed.

Reply:
I was migrating any remaining content from v3.9 ( as stated in the comment under the Mule 4 Doc plan for this doc). Thanks for the update.